themes/custom/droopler_theme/templates/includes/footer.html.twig

```diff
@@ -57,3 +56,6 @@
     </div>
   {% endblock %}
 </footer>
+<div class="after-footer">
```

Comment:
Do wrzucenia na koniec `<footer>`, nie wiem czy potrzebny tu jest nowy DIV, bo on się pojawi nawet jak regionu nie będzie

Reply:
Trzeba też pamiętać żeby dla tego nowego regionu dać klasę `container`

Reply:
ale chwila.. footer_main, to nie jest nowy region.. Piotrek skorzystał z tego co było..

Reply:
chyba, ze masz na mysli zeby dac twig dla regionu i zeby pamietac o .container :thinking: